aspnetcore/client-side/blazor/index.md

```diff
@@ -21,15 +21,15 @@ To learn more about Blazor's technological backbone, see the following documenta
 
 ## Get started
 
-For an overview of Blazor features, see [Blazor at the .NET website](https://www.microsoft.com/net/web/blazor). For development setup instructions and a quick blast of Blazor, see [Get started with Blazor in 10 minutes](https://www.microsoft.com/net/web/blazor/get-started). You can find answers to common questions in the [Blazor FAQ](https://github.com/aspnet/Blazor/wiki/FAQ). To chat live with other rockstars about Blazor, visit [Gitter](https://gitter.im/aspnet/Blazor).
+For an overview of Blazor features, see [Blazor at the .NET website](https://www.microsoft.com/net/web/blazor). For development setup instructions and a quick blast of Blazor, see [Get started with Blazor in 10 minutes](https://www.microsoft.com/net/web/blazor/get-started). You can find answers to common questions in the [Blazor FAQ](https://github.com/aspnet/Blazor/wiki/FAQ). To chat live with other rockstars about Blazor, visit [Gitter](https://gitter.im/aspnet/Blazor) or [Slack](https://aspnetcore.slack.com/messages/C958WJP2N/) ([signup](http://tattoocoder.com/aspnet-slack-sign-up/)).
```

Comment:
I'd like to shorten this to focus only on getting people to the getting started guide. Could we just have a single link here? Something like:

[Get started with Blazor in less than 10 minutes](https://www.microsoft.com/net/web/blazor/get-started)